src/transform/mod.rs

```diff
@@ -71,6 +71,29 @@ pub enum TxType {
   H_FLIPADST = 15,
 }
 
```

Comment:
That this function is fully covered by our basic encode-decode tests suggests either:
- Coverage is misleading
- The content in our tests is resulting in RDO decisions that hide potential desyncs

https://coveralls.io/builds/33004278/source?filename=src/transform/mod.rs#L74

Reply:
Do you mean coverage of the uv_tx_size branch, or of the match arms? I'd be a bit suspicious of the latter being accurate.